instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
@smarnach I know, but attaching it to `InstanceReference` requires only two lines of code and no migrations, since it works with the way django permissions were designed.

By "global model" I meant "apex model" - more specifically, I meant that "InstanceReference" is the primary model of the `instance` app, and there is only ever that one model serving that purpose. Other models in the instance app like `Instance` (which is abstract), `OpenEdXInstance`, `OpenEdXAppServer` are merely themselves instances of a general pattern and there could be many more such models like `DaliteInstance`, `DaliteAppServer`, etc. So it definitely does not make any sense to attach the permission to any other model in the `instance` app other than `InstanceReference`, if we must attach it to a model.

> The custom model is just a proxy model used to encapsulate creating the content type you need for a permission.
Ah, I see now. I found [the explanation](http://stackoverflow.com/questions/13932774/how-can-i-use-django-permissions-without-defining-a-content-type-or-model) on StackOverflow. I think I should clarify something though: it looks to me like the real purpose of the proxy model is to prevent django from thinking the content type is stale and deleting it. If all you needed was a custom content type, you could simply create one in a single line within a data migration; the proxy model is a roundabout way of doing that.